standalone-node/docs/prepare_bootable_usb_windows.md

```diff
@@ -62,34 +94,66 @@ This document explains the procedure to create a bootable USB device for Standal
    ```shell
    usbipd bind --force --busid <busid for USB Drive>
    ```
-4. Start Ubuntu by running the command in PowerShell:
+4. Start Ubuntu:
 
-   ```shell
-   ubuntu2204.exe
-   ```
-   It will ask for the username and password you set previously. Upon successful login, the Ubuntu terminal will open.
+   a. (WSL) Run the command in PowerShell:
 
-5. Use PowerShell to attach the drive:
+      ```shell
+      ubuntu2204.exe
+      ```
+      It will ask for the username and password you set previously. Upon successful login,
+      the Ubuntu terminal will open.
 
-   ```shell
-   usbipd attach --wsl --busid <busid for USB Drive>
-   ```
+   b. (Hyper-V) Run Hyper-V Manager, right click your VM, and select *Connect...*.
+      Select *Start*.
+
+
+5. Attach the drive:
+
+   a. (WSL) Use PowerShell and run the command:
+
+      ```shell
+      usbipd attach --wsl --busid <busid for USB Drive>
+      ```
+
+   b. (Hyper-V) Start the Ubuntu terminal:
+
+
+      - Install Usbip tools:
+
+        ```bash
+        sudo apt install linux-tools-virtual hwdata
+        sudo update-alternatives --install /usr/local/bin/usbip usbip `ls /usr/lib/linux-tools/*/usbip | tail -n1` 20
+        sudo modprobe vhci-hcd
+        ```
+
+      - Run the following command:
+
+        ```bash
+        usbip attach --remote=<IPv4 address of host> --busid=<busid for USB Drive>
+        ```
+
+   **Now, the USB drive will be mounted in Ubuntu.**
+
+   > **Note**:
+   >
+   > * If the system reboots, the USB device attachment is lost. You can either
+   >   re-attach it after Ubuntu has restarted, or configure the OS to
+   >   [automatically re-attach the device](https://wiki.archlinux.org/title/USB/IP#Binding_with_systemd_service).
+   > * To quickly obtain the IPv4 address of your host machine, start *Command Prompt*,
+   >   and run the following command:
+   >
+   >   ```shell
+   >   ping <name-of-your-computer> -4
```

Comment:
Consider also mentioning `ipconfig` as an alternative, which returns all IP addresses and can be more direct.